src/site/antora/modules/ROOT/examples/manual/markers/log4j2.xml

```diff
@@ -15,25 +15,29 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<Configuration xmlns="https://logging.apache.org/xml/ns" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<Configuration xmlns="https://logging.apache.org/xml/ns"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                xsi:schemaLocation="
                    https://logging.apache.org/xml/ns
                    https://logging.apache.org/xml/ns/log4j-config-2.xsd">
-  <Appenders>
+
+  <appenders>
     <Console name="SQL_LOG">
-      <PatternLayout pattern="%d %p %-32marker - %m%n"/>
+      <PatternLayout pattern="%d{HH:mm:ss.SSS} (%marker) %m%n"/>
     </Console>
-  </Appenders>
-  <Loggers>
-    <Root level="INFO"/>
+  </appenders>
+
+  <loggers>
+    <root level="INFO"/>
     <!-- tag::logger[] -->
-    <Logger name="example" level="ALL"><!--1-->
+    <logger name="example" level="ALL"><!--1-->
```

Comment:
The correct capitalization is `Logger`:

https://github.com/apache/logging-log4j2/blob/2e17be4d22f03cbca4a0644106637e3f949d1731/log4j-core/src/main/java/org/apache/logging/log4j/core/config/LoggerConfig.java#L62

I did republish the XSD a couple of weeks ago, but apparently something is wrong with it (not the examples).